BioArchLinux/snapgene-viewer/PKGBUILD

```diff
@@ -0,0 +1,97 @@
+# Maintainer: Bitals <me at bitals dot xyz>
+# Contributor: Matthijs Tadema <M dot J dot Tadema at gmail dot com>
+# Contributor: Lorenzo Gaifas <brisvag at gmail dot com>
+# Contributor: Antony Lee <anntzer dot lee at gmail dot com>
+pkgname=snapgene-viewer
+pkgver=7.2.0
+_pkgver_major=$(cut -d '.' -f 1 <<<"$pkgver")
+_pkgver_major_middle=$(cut -d '.' -f 1-2 <<<"$pkgver")
+pkgrel=1
+pkgdesc='View plasmid maps, annotate features and share sequences (snapgene freeware edition)'
+arch=('x86_64')
+url='https://www.snapgene.com/snapgene-viewer'
+license=('custom')
+depends=('qt6-base'
+         'qt6-webchannel'
+         'qt6-5compat'
+         'qt6-declarative'
+         'qt6-positioning'
+         'qt6-svg'
+         'qt6-webengine'
+         'libsm'
+         'fontconfig'
+         'nspr'
+         'xz'
+         'expat'
+         'glibc'
+         'gcc-libs'
+         'libc++'
+         'libc++abi'
+         'dbus'
+         'krb5'
+         'libxcb'
+         'libxkbcommon-x11'
+         'xcb-util-image'
+         'xcb-util-wm'
+         'libx11'
+         'libxkbfile'
+         'xcb-util-keysyms'
+         'xcb-util-renderutil'
+         'nss'
+         'hicolor-icon-theme'
+         'libice'
+         'libglvnd'
+         'libxext'
+         'openssl-1.1'
+         'zlib'
+         'qca-qt6'
+         'freetype2'
+         'libxkbcommon'
+         'sh'
+)
+makedepends=('patchelf')
```

Comment:
You don't have build functions, why you use makedepends?

Reply:
Fix rpath using patchelf in the end of package()

Reply:
how about creating a install file?

Reply:
> how about creating a install file?

Not good. Install scripts are run on installation, not packaging.